keyboards/shorty/keymaps/default/keymap.c

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
+    [0] =   { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_MS_WH_DOWN,KC_MS_WH_UP)  }
+};
+#endif
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_A,             KC_D,    
+        KC_P7,   KC_P8,   KC_P9,    
+        KC_P4,   KC_P5,   KC_P6,     
+        KC_P1,   KC_P2,   KC_P3  
+    )
+};
```

Comment:
```suggestion

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [0] = LAYOUT(
        KC_A,             KC_D,    
        KC_P7,   KC_P8,   KC_P9,    
        KC_P4,   KC_P5,   KC_P6,     
        KC_P1,   KC_P2,   KC_P3  
    )
};

#if defined(ENCODER_MAP_ENABLE)
const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
    [0] =   { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(MS_WHLD, MS_WHLU)  }
};
#endif
```

Reply:
This looks unnecessary. What the point of doing this?

Reply:
Convention is to place the encoder map below the keymap.